### ISSUE TITLE
when background is set to transparent, then it becomes a simple pie chart

### ISSUE THREAD

**Issue by tscgist** [dumptyd/vue-css-donut-chart#43]
When using the component, I expect the center circle of a donut to be transparent. So that, when the background property is set to transparent, it lets shine the background through. But becomes a simple pie in this version.

**Comment by matheeusaf** [comment]
I'm facing this issue too, trying to implement a dark theme and try to keep the chart with transparent background but it turns a simple pie. Have you solved this issue?

**Comment by dumptyd** [comment]
See if #1 helps you guys.

**Comment by matheeusaf** [comment]
Nope. Using the default vuetify theme is working without errors. I need the background for ".cdc-overlay" when is dark theme "#1E1E1E", but when i active the dark theme i get this:
![image](https://user-images.githubusercontent.com/22040962/121221806-50054080-c85c-11eb-80c2-acad22a3c18f.png)

![image](https://user-images.githubusercontent.com/22040962/121221776-47146f00-c85c-11eb-8beb-a3fe28da1aa6.png)

**Comment by ryansegus** [comment]
I have the same issue, I would like to achieve a transparent background on the hole so it can look like the screen shots, any progress on this?

![Screenshot 2021-12-06 at 13 15 04](https://user-images.githubusercontent.com/7275531/144844436-d0552112-8de1-4617-a669-30e5e903744d.png)
![Screenshot 2021-12-06 at 13 16 19](https://user-images.githubusercontent.com/7275531/144844528-07d04639-0ee4-4f84-a2a7-876aa217cd07.png)
